README.md

```diff
@@ -14,7 +14,7 @@ It's very easy to get started. Clone this repo, install deps, and start it:
 git clone https://github.com/actualbudget/actual-server.git
 cd actual-server
 yarn install
-yarn start
+yarn dev
```

Comment:
Personally I think this is going to be too confusing to change, or at least not worth the confusion.

I'd prefer a `start:prod` or something that runs just the built sources, and leave `start` alone

Reply:
adjusted.